### DISCUSSION TITLE
EIP-6963 wallet

### DISCUSSION THREAD

**Discussion by swkim109** [WalletConnect/walletconnect-monorepo#3791] [Developer Support] [answered]
Is there any wallet support EIP-6963? I added `EIP6963Connector` to the configuration but Metamask still pops up even though I select Rainbow wallet on the modal.

Could multiple wallets connect to dapp at the same time with `EIP6963Connector`?

**Comment by glitch-txs** [comment, marked as answer]
Hi, there are some wallets already supporting EIP-6963
you can find a list here http://eip6963.org

Connecting with multiple wallets at the same time is a feature we'll eventually add together with Wagmi v2 release.
